tests/modeladmin/test_checks.py

```diff
@@ -1309,3 +1309,29 @@ class BandAdmin(ModelAdmin):
             'custom_permission_action action.',
             id='admin.E129',
         )
+
+    @classmethod
+    def get_action(cls, name=None):
+        def action(modeladmin, request, queryset):
+            pass
+
+        if name:
+            action.__name__ = name
+
+        return action
+
+    def test_check_actions_uniqueness_invalid(self):
+        class BandAdmin(ModelAdmin):
+            actions = (self.get_action(), self.get_action())
+
+        self.assertIsInvalid(
+            BandAdmin, Band,
+            '__name__ attributes of actions defined in BandAdmin must be unique.',
+            id='admin.E130',
+        )
+
+    def test_check_actions_uniqueness_valid(self):
+        class BandAdmin(ModelAdmin):
+            actions = (self.get_action(), self.get_action("fix_this"))
```

Comment:
Please us single quotes.

Reply:
`fix_this` is misleading, because there is nothing to fix here.